site/make.bat

```diff
@@ -55,8 +55,16 @@ if errorlevel 9009 (
 	echo.to the full path of the 'sphinx-build' executable. Alternatively you
 	echo.may add the Sphinx directory to PATH.
 	echo.
-	echo.If you don't have Sphinx installed, grab it from
-	echo.http://sphinx-doc.org/
+	echo.If you don't have Sphinx installed, run
```

Comment:
Seems unrelated to SIMD but fine to include in same change i guess.

Reply:
Yeah, that is a bit of a drive-by